### PR TITLE
US132941 [Adoption] Add Tool Usage Trend by Tool Stacked Area Chart Tools Usage

### PR DESCRIPTION
**Pull request by Vitalii-Misechko**
[US132941](https://rally1.rallydev.com/#/?detail=/userstory/612185838529&fdp=true): [Adoption] Add Tool Usage Trend by Tool Stacked Area Chart to Tools Usage

Demo uses the same list of tools as backend API. Make legend more compact

* Functional Testing 
  * [x] Lit 2 Phase 1 compliant changes
  * [x] Browsers (Chrome, FF, Edge)
  * [x] Accessibility
  * [x] RTL
  * [x] Responsive
* Security, devops, perf: N/A
* UX and docs

### CODE REVIEW

File: components/chart-legend.js [hunk 1]
```diff
@@ -56,17 +56,17 @@ class ChartLegend extends SkeletonMixin(Localizer(MobxLitElement)) {
...
 						width: auto;
 					}
+
+					.d2l-insights-chart-legend-item {
+						width: 160px;
+					}
 				}
 				@media only screen and (min-width: 1268px) {
-					.d2l-insights-chart-legend-item:nth-child(4n) {
+					.d2l-insights-chart-legend-item:nth-child(5n) {
```
Comment (bemailloux) [inline components/chart-legend.js:69]: This seems like it will need to change often

**Review by johngwilkinson — APPROVED** [review]

**Review by bemailloux — APPROVED** [review]
LGTM!

File: components/chart-legend.js (hunk 1 above)
Comment (Vitalii-Misechko) [reply components/chart-legend.js:69]: Could you elaborate, please?

Comment (bemailloux) [reply components/chart-legend.js:69]: Oh I see never mind - every 5th legend item has margin applied. I thought it was just the 5th one only :)

**Pull request merged**